### DISCUSSION TITLE
I can't use FabricToolTags

### DISCUSSION THREAD

**Discussion by 12122234** [FabricMC/fabric-api#2516] [Mod Dev Support] [answered]
Today I wanted to make my block breakable most efficiently by a pickaxe, but I can't use the FabricToolTags tag to do it

**Comment by Technici4n** [comment, marked as answer]
You can add your block to the `#minecraft:mineable/pickaxe` tag for this.

**Reply by 12122234** [reply]
how?

**Reply by 12122234** [reply]
Wait I think I know how

**Reply by 12122234** [reply]
I did it thanks for help

**Reply by ghost** [reply]
ducuop comment ? j'ai le même problème mais je n'arrive pas à le régler

**Reply by Technici4n** [reply]
Regarde comment ajouter des blocs à un tag sur le wiki.
